### ISSUE TITLE
Altdorf Siece Healing behind portal abuse.

### ISSUE THREAD

**Issue by herkboy** [WarEmu/WarBugs#15736]
Altdorf at champion stage. 

20 - Feb - 2020

While wining Stage 3. 

The last (and only one alive) was an ironbreaker (sword and board spec) champion. He stayed alive while healers and him were **abusing** the fact that they could heal him behind the spawning portal so we could not kill the last champion although he was the last alive. (He basically had all his healers free to heal a champion tank. I am even suspecting some guard on him as well behind the portal but i cant prove that) That ended up all losing Champion Status (ours were all alive) after a quite big (10-20 minutes didn't actually take a timer) amount of time. This ended up us losing the fight. 

Sometimes a few people tried to get out and fight but the healers focusing on the tank were free to heal from inside. 

Posting pictures

![abuse 22](https://user-images.githubusercontent.com/54741810/74920639-290bcc80-53d5-11ea-8a5e-f14636761769.jpg)

![abuse](https://user-images.githubusercontent.com/54741810/74920659-3032da80-53d5-11ea-9825-e3d316ec6861.jpg)

**Comment by herkboy** [comment]
![123123123123](https://user-images.githubusercontent.com/54741810/74921876-0975a380-53d7-11ea-9e5a-774f157995d7.png)

**Comment by Carthage123** [comment]
Yeah surely this needs to change. Yes mistakes were made at the end and contributed to our loss, but the huge advantage of wrecking their champs early and then being denied the use of ours on the lord, not to mention their morale build up before their push into the room with the lords cant be working as intended.

**Comment by Tesq** [comment]
same goes for morales, they can be build behind barrier
